### ISSUE TITLE
Acknowledge Yubi Key tap

### ISSUE THREAD

**Issue by corkrean** [gravitational/teleport#20781]
**What would you like Teleport to do?**
Since the Tap any security key prompt can occasionally take a second to proceed to the server, Telport could add a simple message of "MFA factor received, proceeding",  or something like that to confirm to the user that the MFA tap was successful so they don't accidentally end up waiting only to realize the YubiKey is still blinking (if their variety even blinks)

**Comment by zmb3** [comment]
We'll have a closer look, but initial impression is that there's not much we can do because the latency is inside of libfido.

**Comment by russjones** [comment]
@corkrean We'll have @nklaassen look into this. The plan is to create a debug build and share it with the customer. Once we know the scope of the issue, we'll look into getting it schedule. 

No ETA at the moment.

**Comment by nklaassen** [comment]
I set up a cluster in a different continent to get some latency and ran some quick tests:

1. After an MFA tap during login there is no output or any user feedback for about 8-9 seconds. This is pretty bad UX.
2. I enabled debugging with `tsh -d login ...` and we get logs within about ~1 second of the MFA tap.

So we should easily be able to improve this quite a bit even if we can't log "instantly". I will look into the code and draft a PR adding a message at the earliest point after we detect the tap.

Going forward we should consider reducing the roundtrips during `tsh login`, it seems like there are quite a few now.
